.github/workflows/sample-apps-with-v2-beta.yml

```diff
@@ -0,0 +1,24 @@
+name: Building sample apps with v2-beta action
+
+on:
+  push:
+    branches:
+      - master
+      - patch/**
+      - actions/**
```

Comment:
I added this branch pattern so that if anyone wants to test a workflow with an action, then they can create a branch in the format `actions/kichalla/testing.v3.action` for example